merkle_tree/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "merkle_tree"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+hex-literal = "1.0.0"
+sha256 = "1.6.0"
+sha3 = "0.10.8"
```

Comment:
In general we prefer to avoid pining the patch version of dependencies, more info here: https://semver.org/